<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdcDac</class>
 <widget class="QDialog" name="AdcDac">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>438</width>
     <height>32</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="adClosePB">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>4</y>
      <width>56</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ADC - DAC</string>
    </property>
   </widget>
  </widget>
  <widget class="QELabel" name="qelabel">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>72</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(R)-$(N):SBYTE1</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>72</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_2">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>72</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(R)-$(N):DATA1</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>72</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_3">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>72</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):AI:$(R)-$(N):CH1</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="leadingZero">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_4">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>72</y>
     <width>93</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):CALC:$(R)-$(N):CH1</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="precision">
    <number>4</number>
   </property>
   <property name="useDbPrecision">
    <bool>false</bool>
   </property>
   <property name="leadingZero">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>96</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_5">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>96</y>
     <width>57</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):AI:$(R)-$(N):CH2</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="leadingZero">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>96</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_6">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>96</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(R)-$(N):DATA2</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_7">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>96</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(R)-$(N):SBYTE2</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_8">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>96</y>
     <width>93</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):CALC:$(R)-$(N):CH2</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="precision">
    <number>4</number>
   </property>
   <property name="useDbPrecision">
    <bool>false</bool>
   </property>
   <property name="leadingZero">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>128</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_13">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>128</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(T)-$(N):SBYTE1</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>152</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_14">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>152</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(T)-$(N):SBYTE2</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>128</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_15">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>128</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(T)-$(N):CH1</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>152</y>
     <width>17</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
  <widget class="QELabel" name="qelabel_16">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>152</y>
     <width>52</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(T)-$(N):CH2</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2</string>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
   <property name="format" stdset="0">
    <enum>QELabel::UnsignedInteger</enum>
   </property>
   <property name="radix">
    <UInt>16</UInt>
   </property>
  </widget>
  <widget class="QELineEdit" name="qelineedit">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>128</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):AO:$(T)-$(N):CH1</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2</string>
   </property>
  </widget>
  <widget class="QELineEdit" name="qelineedit_2">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>152</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):AO:$(T)-$(N):CH2</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>52</y>
     <width>73</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Chan Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>52</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SByte</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>52</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Raw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>52</y>
     <width>41</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>52</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Converted</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>120</y>
     <width>421</width>
     <height>64</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="dacCh2PB">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>32</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dacCh1PB">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>8</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>68</y>
     <width>421</width>
     <height>53</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QPushButton" name="adcCh1PB">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>4</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="adcCh2PB">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>24</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QELabel" name="adcCh1Lbl">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>188</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):AI:$(R)-$(N):CH$(C).DESC</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2,C=1</string>
   </property>
   <property name="visible" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QELabel" name="adcCh2Lbl">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>188</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):AI:$(R)-$(N):CH$(C).DESC</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,R=3132,N=2,C=2</string>
   </property>
   <property name="visible" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QELabel" name="dacCh1Lbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>188</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(T)-$(N):CH$(C).DESC</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2,C=1</string>
   </property>
   <property name="visible" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QELabel" name="dacCh2Lbl">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>188</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="variable" stdset="0">
    <string>$(P):LI:$(T)-$(N):CH$(C).DESC</string>
   </property>
   <property name="variableSubstitutions" stdset="0">
    <string>P=CHA,T=4132,N=2,C=2</string>
   </property>
   <property name="visible" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayAlarmState">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame</zorder>
  <zorder>qelabel</zorder>
  <zorder>label_2</zorder>
  <zorder>qelabel_2</zorder>
  <zorder>label_3</zorder>
  <zorder>qelabel_3</zorder>
  <zorder>qelabel_4</zorder>
  <zorder>label_4</zorder>
  <zorder>qelabel_5</zorder>
  <zorder>label_5</zorder>
  <zorder>qelabel_6</zorder>
  <zorder>qelabel_7</zorder>
  <zorder>qelabel_8</zorder>
  <zorder>label_6</zorder>
  <zorder>qelabel_13</zorder>
  <zorder>label_7</zorder>
  <zorder>qelabel_14</zorder>
  <zorder>label_8</zorder>
  <zorder>qelabel_15</zorder>
  <zorder>label_9</zorder>
  <zorder>qelabel_16</zorder>
  <zorder>qelineedit</zorder>
  <zorder>qelineedit_2</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>adcCh1Lbl</zorder>
  <zorder>adcCh2Lbl</zorder>
  <zorder>dacCh1Lbl</zorder>
  <zorder>dacCh2Lbl</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
